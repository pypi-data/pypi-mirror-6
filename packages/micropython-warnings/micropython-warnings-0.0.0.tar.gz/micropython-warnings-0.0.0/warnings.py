# MicroPython doesn't support warnings
