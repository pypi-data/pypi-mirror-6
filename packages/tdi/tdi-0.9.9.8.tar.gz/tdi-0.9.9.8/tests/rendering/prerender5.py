#!/usr/bin/env python
import warnings as _warnings
_warnings.resetwarnings()
_warnings.filterwarnings('error')

from tdi.model_adapters import RenderAdapter
from tdi import html

template = html.from_string("""
<anode tdi="level1">
    <node tdi="-nested">
        <xnode tdi="a"></xnode>
    </node>
    <ynode tdi="-:nested">lalala</ynode>
</anode>
""".lstrip())

class Model(object):
    def render_nested(self, node):
        node.repeat(self.repeat_nested, (1, 2))

    def repeat_nested(self, node, item):
        pass


model = Model()
template.render(model, adapter=RenderAdapter.for_prerender)
