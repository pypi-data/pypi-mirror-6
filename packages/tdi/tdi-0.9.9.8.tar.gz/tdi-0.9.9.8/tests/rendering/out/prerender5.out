<anode tdi:scope="=+" tdi="+level1">
    
        <xnode tdi:scope="=+" tdi="+a"></xnode>
    lalala
        <xnode tdi:scope="=+" tdi="+a"></xnode>
    
    
</anode>
